playbooks/group_vars/postgres/public.yml

```diff
@@ -11,6 +11,9 @@ TARSNAP_BACKUP_POST_SCRIPT: "/usr/local/sbin/backup-post.sh"
 TARSNAP_BACKUP_FOLDERS: "{{ POSTGRES_SERVER_BACKUP_DIR }}"
 TARSNAP_ARCHIVE_NAME: "{{ POSTGRES_SERVER_DOMAIN }}"
 
+# Run every 3 hours.
+TARSNAP_BACKUP_HOUR: '0,3,6,9,12,15,18,21'
```

Comment:
How about using `*/3` instead?  We wouldn't control the exact times of the backup anymore, but I don't think we need to.